java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -247,4 +247,21 @@ public void set_missing_value_and_returnOldValue_is_null(BaseClient client) {
         String data = client.set("another", ANOTHER_VALUE, options).get();
         assertNull(data);
     }
+
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void sadd_and_srem(BaseClient client) {
```

Comment:
I would try to use randomized keys. If you were to implement all 4 commands you can copy over the functionality of IT tests in node.